Marlin/ubl_G29.cpp

```diff
@@ -917,15 +924,18 @@
     ubl.has_control_of_lcd_panel = true;
     ubl.save_ubl_active_state_and_disable();   // we don't do bed level correction because we want the raw data when we probe
 
-    SERIAL_PROTOCOLLNPGM("Place Shim Under Nozzle and Perform Measurement.");
     do_blocking_move_to_z(in_height);
-    do_blocking_move_to_xy((float(X_MAX_POS) - float(X_MIN_POS)) / 2.0, (float(Y_MAX_POS) - float(Y_MIN_POS)) / 2.0);
+    do_blocking_move_to_xy((float(UBL_MESH_MAX_X) - float(UBL_MESH_MIN_X)) / 2.0, (float(UBL_MESH_MAX_Y) - float(UBL_MESH_MIN_Y)) / 2.0);
```

Comment:
Slightly more concise:
```cpp
do_blocking_move_to_xy(float(UBL_MESH_MAX_X - (UBL_MESH_MIN_X)) / 2.0, float(UBL_MESH_MAX_Y - (UBL_MESH_MIN_Y)) / 2.0);
```
This also accomplishes the same with no casting needed:
```cpp
do_blocking_move_to_xy(0.5 * (UBL_MESH_MAX_X - (UBL_MESH_MIN_X)), 0.5 * (UBL_MESH_MAX_Y - (UBL_MESH_MIN_Y)));
```